assets/src/stories-editor/blocks/amp-story-page/edit.js

```diff
@@ -275,6 +321,12 @@ class PageEdit extends Component {
 					</PanelColorSettings>
 					<PanelBody title={ __( 'Background Media', 'amp' ) }>
 						<>
+							{
+								isExcessiveVideoSize &&
+								<Notice status="error" isDismissible={ false } >
```

Comment:
Let's use `warning` instead of `error`, since it will still work.

Reply:
Good idea, applied with https://github.com/ampproject/amp-wp/pull/2509/commits/4f262f8a3743c53cbbf64ecd70d03dd25c11fdff.